File: cs/src/formulas/trigonometry.cs
using System;

namespace NaesungMath.CS.Formulas {
    public static class Trigonometry {
        public static double SinDeg(double degrees) {
            return Math.Sin(degrees * Math.PI / 180.0);
        }
        public static double CosDeg(double degrees) {
            return Math.Cos(degrees * Math.PI / 180.0);
        }
        public static double TanDeg(double degrees) {
            return Math.Tan(degrees * Math.PI / 180.0);
        }
    }
}

File: java/src/main/java/com/naesungmath/formulas/Trigonometry.java
package com.naesungmath.formulas;

public class Trigonometry {
    public static double degreeToRad(double degrees) {
        return degrees * (Math.PI / 180.0);
    }

    public static double radian(double a) {
        return degreeToRad(a);
    }

    public static double radToDegree(double radians) {
        return radians * (180.0 / Math.PI);
    }
}
